mbed_greentea/cmake_handlers.py

```diff
@@ -122,3 +122,13 @@ def list_binaries_for_builds(test_spec, verbose_footer=False):
     if verbose_footer:
         print
         print "Example: execute 'mbedgt -t BUILD_NAME -n TEST_NAME' to run test TEST_NAME for build TARGET_NAME in current test specification"
+
+def list_test_cases_for_binaries(test_spec):
+    """! Parse test spec and list test cases inside the binaries
+    @param test_spec Test specification object
+    """
+    test_cases = test_spec.get_test_cases_by_binary()
+    for binary, tc in test_cases.iteritems():
+        gt_logger.gt_log("available test cases for test binary '%s'"% binary)
+        for case in tc:
```

Comment:
@mazimkhan This dict is populated in the TestSpec class, and so does not have that form. The method that produces this dict can be found [here](https://github.com/ARMmbed/greentea/pull/175/files#diff-269c77b7b96e1fbece62d10d0484dc80R385).